.ki-scripts/README.md

```diff
@@ -0,0 +1,45 @@
+# Kitty Items Scripts
+
+## Version Check
+
+A script that compares the current process's Node version against the [package.json](/package.json) Node engine requirement.
+
+## Startup
+
+This script is called when deploying both local & testnet versions of the Kitty Items application. Multiple processes are executed to initialize both the database as well as an admin service account required for the application to function.
+
+### Begin Process
+The script begins by ending all running processes from previously launched applications.
+
+### Check for correct Node version
+In this step, the script checks the process's Node version against the Node engine requirement in the root [package.json](/package.json). Similar to [version-check.js](/.ki-scripts/version-check.js).
+
+### Testnet account creation (`bootstrapNewTestnetAccount()`)
+This section defines the function that begins the process of creating a new Testnet account for applications running on the testnet environment. This account becomes the admin service account that eventually mints your Kitty Items on the testnet.
```

Comment:
```suggestion
This section defines the function that begins the process of creating a new Testnet account for applications that connect to and run Cadence on the testnet. This account becomes the admin service account that eventually mints your Kitty Items on the testnet.
```

Reply:
I'm still not sure whether to use "the testnet" or Testnet ... but I think we should pick one per doc and stick to it, instead of having both. This is a minor complaint, but it bugs me.

Reply:
going with Testnet